foundations/javascript_basics/fundamentals-5.md

```diff
@@ -34,6 +34,8 @@ At this point you just need a little more practice!  Go back to the [JavaScript
 - getTheTitles
 - findTheOldest
 
+Don't remember how to do them? No worries! [Here](https://www.theodinproject.com/lessons/foundations-fundamentals-part-4#practice) you have a refresher on how to work with Node.js
```

Comment:
I feel the term "how to do them" might be vague and misleading, as they might imply that the subsequent link takes them to the solution. This is just how I interpret it. We could be more specific about how we word this sentence. Furthermore, "Here" is not an adequate link text for accessibility reasons. 

I have a suggested change below, feel free to edit wherever you think is needed.
```suggestion
Don't remember how to setup and test the exercises? No worries! Check out the [Fundamentals Part 4 lesson](https://www.theodinproject.com/lessons/foundations-fundamentals-part-4#practice) for a refresher.
```

Reply:
I think this is perfect. I was looking for someone to correct me because I was sure I was missing something, but I just couldn't see it.
I totally agree with the wording!